### PR TITLE
RHDEVDOCS-3619: Update the creating applications flow as per the latest UI changes

### PR DESCRIPTION
**Pull request by HarshCasper**
- JIRA ID: [RHDEVDOCS-3619](https://issues.redhat.com/browse/RHDEVDOCS-3619)
- Aligned team: Dev Tools
- SME review: @jerolimov 
- Peer Review: Pending
- Docs Preview: https://deploy-preview-41972--osdocs.netlify.app/openshift-enterprise/latest/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#odc-importing-codebase-from-git-to-create-application_odc-creating-applications-using-developer-perspective

This PR is for the 4.10 release.

### CODE REVIEW

File: modules/odc-importing-codebase-from-git-to-create-application.adoc
Comment (jerolimov) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:31]: We call it "Node.js application" above, and based on their website this is how Node.js calls theirself.

```suggestion
.. Optional: For Node.js builder image, use the **Run command** field to override the command to run the application.
```

File: modules/odc-importing-codebase-from-git-to-create-application.adoc
Comment (jerolimov) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:55]: The target port doesn't change the external "public available URL" port. It changes the port where the application (container) exposes their data and which will be then connected/proxied by the Service.

Based on your change I would recommend something like this:

```suggestion
. In the *Advanced Options* section, the *Target port* and the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. You can change the target port if your application uses a different port to expose their data. You can clear the check box if you do not want to expose your application on a public route.
```

**Review by christoph-jerolimov — CHANGES_REQUESTED** [review]
Just small suggestions:

**Review by christoph-jerolimov — APPROVED** [review]
Technically this is fine :+1: 

/lgtm

File: modules/odc-importing-codebase-from-git-to-create-application.adoc
Comment (rolfedh) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:55]:
```suggestion
. Optional: In the *Advanced Options* section, the *Target port* and the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. This URL uses port [TBD? 80?] by default. 
+
If your application does not expose its data on the default public port, 80, clear the [name of] check box, and set the target port number by [doing TBD]
```
I don't know the operation, but I have the impression it might help to expand it a bit more. This suggestion is a pure guess of what this might look like. What do you think?

Comment (rolfedh) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc:55]: Otherwise: 
- It looks like you might be using "their" as a gender-neutral possessive for the application, a single inanimate object. Is this anthropomorphism? How about using "its" instead? 
- If the user makes a choice based on a condition, "If...then", does putting the "if" condition first make that choice easier for the user to understand?

**Review by rolfedh — COMMENTED** [review]
Nice work, @HarshCasper. Left some suggestions. Ping me when you need me to take another look. Thanks!

File: modules/odc-importing-codebase-from-git-to-create-application.adoc
Comment (rolfedh) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:57]: How does one set the target port number? By appending it to the URL, or is there a field where one must enter the port number?

**Review by rolfedh — APPROVED** [review]
Nice work, @HarshCasper. Without the interface in front of me, it's unclear whether it makes sense to give the user more specific instructions on setting the non-default port number. I leave it up to you...

**Comment by rolfedh** [comment]
Please remember to fetch, rebase, and rebuild locally without errors before requesting merge.

File: modules/odc-importing-codebase-from-git-to-create-application.adoc
Comment (HarshCasper) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc:57]: There is a field where one must enter the port number.

Comment (Preeticp) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:57]: ...and set the target port number as the port number you want to expose.
Will the technical meaning of the above part change if we wrote this as: ...and set the target port number you want to expose.

File: modules/odc-importing-codebase-from-git-to-create-application.adoc
Comment (Preeticp) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:88]: 'a toast notification is displayed' - the description of the type of notification is a background UI detail that does not add much value to the user. What would make sense to the user is something like, 'a success notification is displayed'.
Now that it is a separate sentence, suggest: You can see its build status.../You can see the build status of the application...

File: modules/odc-importing-codebase-from-git-to-create-application.adoc
Comment (HarshCasper) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc:57]: Nope, it won't.

**Pull request merged**
